naver-searchad-api.gemspec

```diff
@@ -7,7 +7,7 @@ Gem::Specification.new do |spec|
   spec.name          = 'naver-searchad-api'
   spec.version       = Naver::Searchad::Api::VERSION
   spec.authors       = ['Min Kim']
-  spec.email         = ['developers@forward3d.com', 'min.kim@forward3d.com']
+  spec.email         = %w[developers@forward3d.com min.kim@forward3d.com]
```

Comment:
here you are using % literal with `%[]` but after you are using `%{}`, I think we should be consistent :)